lib/src/tasks/analyze/config.dart

```diff
@@ -19,9 +19,11 @@ import 'package:dart_dev/src/tasks/config.dart';
 const List<String> defaultEntryPoints = const ['lib/'];
 const bool defaultFatalWarnings = true;
 const bool defaultHints = true;
+const bool defaultFatalHints = false;
 
 class AnalyzeConfig extends TaskConfig {
   List<String> entryPoints = defaultEntryPoints.toList();
   bool fatalWarnings = defaultFatalWarnings;
   bool hints = defaultHints;
+  bool failOnHints = defaultFatalHints;
```

Comment:
"fatalHints" is used everywhere else, I think the config option should be consistent with that